pkg/node/node_address.go

```diff
@@ -203,24 +214,36 @@ func Uninitialize() {
 	ipv6AllocRange = nil
 }
 
-// SetNodePortIPv4 sets the node-port IPv4 address for NAT
-func SetNodePortIPv4(ip net.IP) {
-	ipv4NodePortAddress = ip
+// GetNodePortIPv4 returns the node-port IPv4 address for NAT
+func GetNodePortIPv4Addrs() []net.IP {
+	addrs := make([]net.IP, 0, len(ipv4NodePortAddrs))
+	for _, addr := range ipv4NodePortAddrs {
+		addrs = append(addrs, addr)
+	}
+	return addrs
 }
 
-// SetNodePortIPv6 sets the node-port IPv6 address for NAT
-func SetNodePortIPv6(ip net.IP) {
-	ipv6NodePortAddress = ip
+// GetNodePortIPv4AddrsWithDevices returns the map iface => NodePort IPv4.
+//
+// NOTE: Do not modify the returned map.
+func GetNodePortIPv4AddrsWithDevices() map[string]net.IP {
+	return ipv4NodePortAddrs
```

Comment:
why does this function have this comment but `GetNodePortIPv4Addrs()` was copied before returning?a